_source/_posts/2019-03-13-build-rest-api-with-aspnet-web-api.md

```diff
@@ -261,26 +261,28 @@ Here's how it works:
 
 (If this is all new to you, you can [learn more about OAuth here](https://developer.okta.com/blog/2017/06/21/what-the-heck-is-oauth).)
 
-You probably don't want to build an OAuth 2.0 authorization server yourself, so for this tutorial, you will sign up for a [forever-free Okta developer account](https://developer.okta.com/signup/) (or sign in to `{yourOktaDomain}` if you already have an account).
+You probably don't want to build an OAuth 2.0 authorization server yourself, so for this tutorial, you will sign up for a forever-free Okta developer account; or sign in to `{yourOktaDomain}` if you already have an account). Install the [Okta CLI](https://cli.okta.com/) and run `okta register` to sign up for a new account. If you already have an account, run `okta login`.
```

Comment:
You should be able to replace this whole section with:

```
{% include setup/cli.md type="service" %}
```

If there are improvements you need to make when type == service, please feel free to update `cli.md`.

Reply:
Updated